dvc/command/remove.py

```diff
@@ -4,6 +4,7 @@
 from dvc.command import completion
 from dvc.command.base import CmdBase, append_doc_link
 from dvc.exceptions import DvcException
+from dvc.stage.exceptions import StageNotFound
```

Comment:
Note that importing internal modules in `dvc/command` costs us the startup time. If this is needed, it could be imported in `Cmd::run()` below. But, probably it's not required.

Reply:
[offtopic]
I've fixed that. What is the process of reviewing such changes? Should I press "Resolve Conversion"? Will the changes automatically be displayed here? Are there any other actions required by my side?
[/offtopic]